defaults.py

```diff
@@ -24,4 +24,11 @@
 }
 TIMEZONE_SHIFT = datetime.timedelta(hours=0)  # Thanks Whatsapp
 
-HEBREW_LETTERS = "[אבגדהוזחטיכלמנסעפצקרשתךםןףץ]"
\ No newline at end of file
+HEBREW_LETTERS = "[אבגדהוזחטיכלמנסעפצקרשתךםןףץ]"
+
+ANDROID_DATES = r'(\d{2}\/\d{2}\/\d{4}\, \d{1,2}:\d{2}) \- (.*?): (.*)$'

```

Comment:
We need to check if this also covers phones who use AM/PM notations on hours - I suspect it won't, and that we will need to devise some method to differentiate

Reply:
Agree. Adding as separate issue. I will need to do experiments with the settings to see if it changes the text retrospectively.